learn/pkg/learn/renderer.go

```diff
@@ -206,23 +251,30 @@ func newRendererFromEvyFile(filename string, resultType ResultType) (Renderer, e
 	if err != nil {
 		return nil, err
 	}
+	if strings.HasSuffix(filename, ".txtar") {
+		return newTxtarContent(filename, resultType)
+	}
```

Comment:
This could go before reading all the bytes so you dont read them twice.

Reply:
Good point! thank you. fixed.